artifacts/rhdh-plugins-reference/argocd/argocd-plugin-readme.adoc

```diff
@@ -1,322 +1,114 @@
 
-The error handling in ArgoCD is designed to alert you when proper permissions are not in place.
-
-=== Self-signed certificates in ArgoCD plugin
-
-By default, the ArgoCD server generates a self-signed certificate. Once you install the ArgoCD plugin, the deployment of argocd-server can be patched to be insecure using the following command:
-
-[source,console]
-----
-kubectl patch deployment argocd-server --type "json" -p '[{"op":"add","path":"/spec/template/spec/containers/0/command/-","value":"--insecure"}]'
-----
-
-Also, you can use Helm to install the ArgoCD plugin and be insecure by default:
-
-[source,console]
-----
-helm upgrade --install argocd argo/argo-cd \
-  --version 3.33.5 \
-  --set 'server.extraArgs={--insecure}'
-----
-
-== For users
-
-=== Using ArgoCD plugin in {product}
-
-The ArgoCD plugin displays the current state of an application in Backstage Catalog.
-
-.Prerequisites
-
-* Your Backstage application is installed and running.
-* You have installed the ArgoCD plugin. For installation steps, see <<Installation>>.
-
-.Procedure
-
-. Open your Backstage application and go to the *Catalog* page.
-. In the *OVERVIEW* tab, you can see the ArgoCD overview card at the bottom.
 +
---
+image::rhdh-plugins-reference/sidebar.png[Sidebar]
 
-image::argo-cd-plugin-overview-card.png[ArgoCD overview card]
-
-The ArgoCD overview card displays application information including *NAME*, *INSTANCE*, *SYNC STATUS*, *HEALTH STATUS*, and *LAST SYNCED*.
---
-
-. To view the detailed application information, select an application from the ArgoCD overview card.
-+
---
-A pop-up containing detailed information about the application appears, along with a link to ArgoCD dashboard.
++ 
+* Select the link icon (image:rhdh-plugins-reference/link.svg[width=13px, height=13px, align=center]) to open the deployment details in Argo CD.
 
-image::argo-cd-plugin-overview-card-details.png[ArgoCD overview card]
---
+. Select the *Overview* tab and navigate to the Deployment summary section to review the summary of your application's deployment across namespaces. Additionally, select an appropriate Argo CD app to open the deployment details in Argo CD, or select a commit from the Revision column to review the changes in GitLab or GitHub.
 
-. To view the ArgoCD history, go to the *CD* tab in the Catalog.
 +
---
-
-image::argo-cd-plugin-history.png[ArgoCD overview card]
-
-The *ArgoCD history* page contains history of application instances along with other information, such as deploy details, author name, message that author added, and revision ID.
---
-
-
-
-
+image::rhdh-plugins-reference/deployment_summary.png[Deployment summary]
 
 
+[role="_additional-resources"]
+.Additional resources
 
+* For more information on dynamic plugins, refer link:https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.2/html/configuring_plugins_in_red_hat_developer_hub/rhdh-installing-dynamic-plugins[Dynamic plugin installation].
```

Comment:
Change _refer_ to _see_ to align with the rest of the RHDH docs. It also looks like "to" is missing after "refer" for _refer to_